strider/trapi.py

```diff
@@ -283,3 +359,23 @@ def add_descendants(
     })
 
     return qg
+
+
+def build_unique_kg_edge_ids(message: Message):
+    """
+    Replace KG edge IDs with a string that represents
+    whether the edge can be merged with other edges
+    """
+    for edge_id in list(message["knowledge_graph"]["edges"].keys()):
+        edge = message["knowledge_graph"]["edges"].pop(edge_id)
+        new_edge_id = f"{edge['subject']}-{edge['predicate']}-{edge['object']}"
```

Comment:
In my experience, long edge ids like this make it annoying for this human to read a message. Would we lose anything by using a short hash instead? Aside from, theoretically, collisions.
```suggestion
        new_edge_id = hashlib.blake2b(f"{edge['subject']}-{edge['predicate']}-{edge['object']}".encode(), digest_size=6).hexdigest()
```
For 100,000 edges and a 6-byte hash, the chance of a collision is on the order of 1e-5, if my calculations are correct.